scripts/run-consumer-harness.py

```
@@ -112,6 +152,139 @@ def current_host_npm_platform() -> str:
+def exercise_python_wheel(path: Path, scratch: Path) -> str:
+    target_os_cpu = wheel_platform_os_cpu(path.name)
+    host_os, host_cpu = current_host_npm_os_cpu()
+    if target_os_cpu is None or target_os_cpu != (host_os, host_cpu):
+        return (
+            f"checksum-verified only ({path.name} targets "
+            f"{target_os_cpu or 'an unrecognized platform tag'}, host is {host_os}/{host_cpu})"
+        )
+
+    venv_dir = scratch / f"{path.stem}-venv"
+    try:
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)